### ISSUE TITLE
SampleWPFTrader login

### ISSUE THREAD

**Issue by ifcquery** [IG-Group/ig-webapi-dotnet-sample#7]
I compiled the SampleWPFTrader application, replaced username and password with my demo account from IG Markets.
The application runs, but connection to IG server fails:
"Failed to login. HttpResponse StatusCode = Forbidden"

Is there anything else I have to change?

Do I have to change the ApiKey? If yes, how can I get one? I did not find anything like that in my demo account on https://www.ig.com/en/myig/

Thanks!

**Comment by andysinclair** [comment]
Yes, you need to create a demo API Key. Some instructions here (step 3):
[https://www.excelpricefeed.com/userguide/account-setup-ig-index](https://www.excelpricefeed.com/userguide/account-setup-ig-index)

**Comment by ifcquery** [comment]
Thanks for the quick reply.

On the website, I looked for the Link to create an API key, but found nothing:

![2019-02-24 13_43_10-my ig](https://user-images.githubusercontent.com/11755247/53299438-55305400-383a-11e9-98a9-32a1c0947b4a.png)

Also under "Settings", there is nothing like "API key"

**Comment by andysinclair** [comment]
I don't understand why you can't see API Keys under settings, maybe you should contact IG Index directly?

![capture](https://user-images.githubusercontent.com/1213190/53302330-d224f300-3865-11e9-8aa3-38d0c6b310ef.PNG)

**Comment by nsupathy** [comment]
You need an API key and that comes with a live account/Public API.  Try contacting the customer services.
